packages/python-sat/test_python_sat.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from pyodide_test_runner import run_in_pyodide
+
+solvers = [
+    "cadical",
+    "gluecard30",
+    "gluecard41",
+    "glucose30",
+    "glucose41",
+    "maplechrono",
+    "maplecm",
+    "maplesat",
+    "mergesat3",
+    "minicard",
+    "minisat22",
+    "minisat-gh",
+]
+
+
+@pytest.mark.parametrize("name", solvers)
+@run_in_pyodide(packages=["python-sat"])
+def test_solvers(selenium, name):
+
+    from pysat.formula import CNF
+    from pysat.solvers import Solver
+
+    cnf = CNF(from_clauses=[[1, 2, 3], [-1, 2], [-2]])
+
+    with Solver(name=name, bootstrap_with=cnf) as solver:
+        solver.solve()
+        stats = solver.accum_stats()
+        assert "conflicts" in stats, f"No conflicts for {name}"
+        # assert 'decisions' in stats, 'No decisions for {0}'.format(name)
+        # assert 'propagations' in stats, 'No propagations for {0}'.format(name)
+        assert "restarts" in stats, f"No restarts for {name}"
+
+
+@run_in_pyodide(packages=["python-sat"])
+def test_atmost(selenium):
+    from pysat.card import CardEnc
+    from pysat.formula import IDPool
+
+    vp = IDPool()
+    n = 20
+    b = 50
+    assert n <= b
+
+    literals = [vp.id(v) for v in range(1, n + 1)]
+    top = vp.top
+
+    G = CardEnc.atmost(literals, b, vpool=vp)
+
+    assert len(G.clauses) == 0
+
+    try:
+        assert vp.top >= top
+    except AssertionError:
+        print(f"\nvp.top = {vp.top} (expected >= {top})\n")
+        raise
+
+
+@run_in_pyodide(packages=["python-sat"])
+def test_atmost1(selenium):
+    from pysat.card import CardEnc, EncType
+    from pysat.solvers import MinisatGH
+
+    encs = list(
+        filter(
+            lambda name: not name.startswith("__") and name != "native", dir(EncType)
+        )
+    )
+    for l in range(10, 20):
+        for e in encs:
+            cnf = CardEnc.atmost(
+                literals=list(range(1, l + 1)), bound=1, encoding=getattr(EncType, e)
```

Comment:
Here it should be `lits=list(range(1, l + 1))` rather than `literals=list(range(1, l + 1))`.